input_tool.py

```diff
@@ -0,0 +1,203 @@
+db_password = os.environ.get('DB_PASSWORD')
+
+def create_tables(cur):
+    # Users table
+    cur.execute("""CREATE TABLE IF NOT EXISTS owners (
+                owner_id serial PRIMARY KEY,
+                first_name VARCHAR,
+                last_name VARCHAR,
+                username VARCHAR NOT NULL UNIQUE
+    )""")
+    # Experiments table
+    cur.execute("""CREATE TABLE IF NOT EXISTS experiments (
+                experiment_id serial PRIMARY KEY,
+                experiment_name VARCHAR,
+                owner_id INTEGER NOT NULL,
+                CONSTRAINT fk_owner
+                    FOREIGN KEY (owner_id)
+                        REFERENCES owners(owner_id)
+                        ON DELETE CASCADE,
+                UNIQUE(experiment_name, owner_id)
+    )""")
+    # Groups table
+    cur.execute("""CREATE TABLE IF NOT EXISTS groups (
+                group_id serial PRIMARY KEY,
+                group_name VARCHAR
+    )""")
+    # Variables table
+    cur.execute("""CREATE TABLE IF NOT EXISTS variables (
+                variable_id serial PRIMARY KEY,
+                variable_name VARCHAR,
+                channel INTEGER
+    )""")
+    # Observations table
+    cur.execute("""CREATE TABLE IF NOT EXISTS observations (
+                observation_id serial PRIMARY KEY,
+                observation_name VARCHAR,
+                variable_id INTEGER NOT NULL,
+                CONSTRAINT fk_variable
+                    FOREIGN KEY (variable_id)
+                        REFERENCES variables(variable_id)
+                        ON DELETE CASCADE      
+    )""")
+    # Plots table
+    cur.execute("""CREATE TABLE IF NOT EXISTS plots (
+                plot_id serial PRIMARY KEY,
+                div VARCHAR,
+                script VARCHAR,
+                experiment_id INTEGER NOT NULL,
+                group_id INTEGER NOT NULL,
+                observation_id INTEGER NOT NULL,
+                CONSTRAINT fk_experiment
+                    FOREIGN KEY (experiment_id)
+                        REFERENCES experiments(experiment_id)
+                        ON DELETE CASCADE,
+                CONSTRAINT fk_group
+                    FOREIGN KEY (group_id)
+                        REFERENCES groups(group_id)
+                        ON DELETE CASCADE,
+                CONSTRAINT fk_observation
+                    FOREIGN KEY (observation_id)
+                        REFERENCES observations(observation_id)
+                        ON DELETE CASCADE
+    )""")
+
+def drop_tables(cur):
+    cur.execute("DROP TABLE IF EXISTS owners CASCADE")
+    cur.execute("DROP TABLE IF EXISTS experiments CASCADE")
+    cur.execute("DROP TABLE IF EXISTS plots CASCADE")
+    cur.execute("DROP TABLE IF EXISTS groups CASCADE")
+    cur.execute("DROP TABLE IF EXISTS observations CASCADE")
+    cur.execute("DROP TABLE IF EXISTS variables CASCADE")
+
+def load_dataset_to_db(cur):
+    with open("dataset.json", 'rb') as dataset:
```

Comment:
How will `dataset.json` be automatically generated? When the input tool is used in the context of swell, it will be given a user, experiment_name, and the location of the eva directory with pkl files